pkg/env/action.go

```diff
@@ -31,21 +31,23 @@ const(
 
 // action a group env functions
 type action struct {
-	role actionRole
-	ctx context.Context
+	role  actionRole
+	ctx   context.Context // nolint
```

Comment:
If I recall, this is not used right now, field `ctx` can be removed.

Reply:
yes, this is not used, but i thought is something that will be needed, that why I added the nolint :) but will remove

Reply:
thanks for your Review!